src/Cake.AzureDevOps/Repos/PullRequest/AzureDevOpsPullRequest.cs

```diff
@@ -809,6 +818,37 @@ public IEnumerable<AzureDevOpsPullRequestIterationChange> GetIterationChanges(in
             }
         }
 
+        /// <summary>
+        /// Sets the pull request comment thread status.
+        /// </summary>
+        /// <param name="threadId">The Id of the comment thread.</param>
+        /// <param name="status">The comment thread status.</param>
+        public void SetCommentThreadStatus(int threadId, CommentThreadStatus status)
```

Comment:
We can't make this public since `CommentThreadStatus` is coming from Azure DevOps API which is considered an implementation detail for this addin.